src/crab/splitdbm/adapt_sgraph.hpp

```
@@ -27,87 +28,61 @@ class TreeSMap final {
     }
 
     struct KeyIterator {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: e

(cppcoreguidelines-pro-type-member-init)